spirv_cross.hpp

```diff
@@ -371,6 +371,9 @@ class Compiler
 	spv::ExecutionModel get_execution_model() const;
 
 	bool is_tessellation_shader() const;
+	bool is_tesc_shader() const;
```

Comment:
These seem like they should be private helpers.

Reply:
I've hoisted them into `CompilerMSL`.